pv_site_production/app.py

```diff
@@ -69,6 +82,14 @@ def main(
 
     if timestamp is None:
         timestamp = datetime.utcnow()
+        if round_date_to_minutes:
```

Comment:
is it worth extracting this in to its own function?

Reply:
I think both (extracted or inline) are fine